### DISCUSSION TITLE
Assistance Requested: Name Still Appears in Contributors After Branch Deletion and PR History

### DISCUSSION THREAD

**Discussion by syafinion** [community/community#120578] [Repositories]
Hello,

I've come across a situation in our GitHub repository that I'm hoping to get some advice on. For personal reasons, I needed to remove my name from the list of contributors. To this end, I have deleted a branch that contained all of my commits. However, I had previously made pull requests that were merged into the main branch. I have not created any forks of this repository.

After deleting my branch, I've observed that my name persists in the Contributors section. There's been a considerable waiting period since this action, and the list does not seem to reflect the changes. 

My questions are:

1. Does GitHub have a known delay when updating the Contributors list after such actions, and how long might this take?
2. Can any steps be initiated from within the repository's settings or otherwise to trigger an update of the Contributors section?
3. Considering the pull requests I made in the past, which have now been merged and the branch deleted, is there a community-endorsed method to have these contributions reassessed for the Contributors list?

Any guidance or similar experiences shared would be incredibly helpful as I navigate this situation. Thank you for your support and understanding.

**Comment by nibazshab** [comment]
Just nee changing the default branch name and restoring the name.
